Backpack/src/main/java/net/skyscanner/backpack/util/BpkViewPumpContextWrapper.kt

```diff
@@ -0,0 +1,19 @@
+package net.skyscanner.backpack.util
+
+import android.content.Context
+import android.content.ContextWrapper
+import io.github.inflationx.viewpump.ViewPumpContextWrapper
+
+object BpkViewPumpContextWrapper {
+  /**
+   * Disables Viewpump on Android Q and above.
+   * See https://github.com/InflationX/ViewPump/issues/21
+   */
+  fun wrap(context: Context): Context {
+    return if (android.os.Build.VERSION.SDK_INT > android.os.Build.VERSION_CODES.P) {
+      ContextWrapper(context)
```

Comment:
Sorry I meant returning the context directly here without wrapping, e.g. just `context` instead of `ContextWrapper`